apps/etterlatte-behandling-kafka/src/main/kotlin/no/nav/etterlatte/tidshendelser/TidshendelseService.kt

```diff
@@ -172,8 +172,7 @@ class TidshendelseService(
                 oppgaveTypeFor(hendelse.jobbtype),
                 referanse = null,
                 merknad = hendelse.jobbtype.beskrivelse,
-                // TODO finne rett frist
-                frist = Tidspunkt.ofNorskTidssone(hendelse.behandlingsmaaned.atEndOfMonth(), LocalTime.NOON),
+                frist = Tidspunkt.ofNorskTidssone(hendelse.behandlingsmaaned.plusMonths(2).atDay(21), LocalTime.NOON),
```

Comment:
Stemmer plusMonths(2), og hva er egentlig behandlingsmåned for denne jobben - er det måneden de fyller 18? For i så fall er det vel heller minusMonths(1) vi vil ha som fristen

Reply:
behandlingsmåned for denne jobben er måneden de fyller 18. Jeg tok utgangspunkt i at kjøredato og behandlingsmåned er samme mnd. `minusMonths(1)` vil jo sette frist på oppgave tilbake i tid i dette tilfelle.

Reply:
Jeg oppfattet det slik at de som fyller 18 i juni burde få oppgaven i april. Hvis behandlingsmåned da blir juni, og de får oppgaven opprettet i april, burde fristen være mai - som da blir juni minus en måned = behandlingsmåned - 1

Reply:
jeg tenker jo at behandlingsmåned = måned de fyller 18 er den enkleste tolkningen for vår del, også for ettertiden.